browser/ui/tabs/split_view_browser_data.cc

```diff
@@ -191,4 +200,30 @@ SplitViewBrowserData::TabDragStarted() {
                 weak_ptr_factory_.GetWeakPtr()));
 }
 
+void SplitViewBrowserData::TabsWillBeAttachedToNewBrowser(
+    const std::vector<tabs::TabHandle>& tabs) {
+  DCHECK(tiles_to_be_attached_to_new_window_.empty());
+
+  base::flat_set<Tile> tiles_to_be_attached_to_new_window;
+  for (const auto& tab : tabs) {
+    auto iter = FindTile(tab);
+    if (iter == tiles_.end()) {
+      continue;
+    }
+
+    tiles_to_be_attached_to_new_window.insert(*iter);
```

Comment:
Not sure if its a concern here because presumably the number of tiles is small but constructing a `base::flat_set` via insertion is O(n^2) - might be worth using a different set?

Reply:
Oops, I always forget about it. Thanks

Reply:
Me too :laughing: